src/mmw/js/src/core/views.js

```diff
@@ -673,6 +679,19 @@ var MapView = Marionette.ItemView.extend({
             }
         });
     },
+
+    renderFocusBounds: function() {
```

Comment:
Removing this "focus bounds" commit because there's a more robust version coming up in the next PR.